server/test/unit/BraveAlerterConfiguratorTest/getReturnMessageToOtherResponderPhoneNumbersTest.js

```diff
@@ -6,83 +6,104 @@ const { before, describe, it } = require('mocha')
-  })
-
-  it('should get message when WAITING_FOR_REPLY => WAITING_FOR_CATEGORY', () => {
-    const returnMessage = this.alertStateMachine.getReturnMessageToOtherResponderPhoneNumbers(
-      'en',
-      CHATBOT_STATE.WAITING_FOR_REPLY,
-      CHATBOT_STATE.WAITING_FOR_CATEGORY,
-      'Selected Category',
-    )
-
-    expect(returnMessage).to.equal(`Another Responder has acknowledged this request. (You don't need to respond to this message.)`)
-  })
-
-  it('should get no message when WAITING_FOR_CATEGORY => WAITING_FOR_CATEGORY', () => {
-    const returnMessage = this.alertStateMachine.getReturnMessageToOtherResponderPhoneNumbers(
-      'en',
-      CHATBOT_STATE.WAITING_FOR_CATEGORY,
-      CHATBOT_STATE.WAITING_FOR_CATEGORY,
-    )
-
-    expect(returnMessage).to.equal(null)
-  })
-
-  it('should get message when WAITING_FOR_CATEGORY => COMPLETED', () => {
-    const returnMessage = this.alertStateMachine.getReturnMessageToOtherResponderPhoneNumbers(
-      'en',
-      CHATBOT_STATE.WAITING_FOR_CATEGORY,
-      CHATBOT_STATE.COMPLETED,
-      'Selected Category',
-    )
-
-    expect(returnMessage).to.equal(
-      `The incident was categorized as Selected Category.\n\nThank you. This session is now complete. (You don't need to respond to this message.)`,
-    )
-  })
-
-  it('should get no message when COMPLETED => COMPLETED', () => {
-    const returnMessage = this.alertStateMachine.getReturnMessageToOtherResponderPhoneNumbers(
-      'en',
-      CHATBOT_STATE.COMPLETED,
-      CHATBOT_STATE.COMPLETED,
-      ['Cat0'],
-    )
-
-    expect(returnMessage).to.equal(null)
-  })
-
-  it('should get default message if given something funky', () => {
-    const returnMessage = this.alertStateMachine.getReturnMessageToOtherResponderPhoneNumbers('en', 'something funky', CHATBOT_STATE.COMPLETED, [
-      'Cat0',
-    ])
-
-    expect(returnMessage).to.equal('Error: No active session found')
+  languages.forEach(language => {
+    it('should get message when STARTED => WAITING_FOR_REPLY', () => {
+      const returnMessage = this.alertStateMachine.getReturnMessageToOtherResponderPhoneNumbers(
+        language,
+        CHATBOT_STATE.STARTED,
+        CHATBOT_STATE.WAITING_FOR_REPLY,
+      )
+      expect(returnMessage).to.equal(null)
+    })
+
+    it('should get message when STARTED => WAITING_FOR_CATEGORY', () => {
+      const returnMessage = this.alertStateMachine.getReturnMessageToOtherResponderPhoneNumbers(
+        language,
+        CHATBOT_STATE.STARTED,
+        CHATBOT_STATE.WAITING_FOR_CATEGORY,
+      )
+
+      if (language === 'en') {
+        expect(returnMessage).to.equal(`Another Responder has acknowledged this request. (You don't need to respond to this message.)`)
+      } else {
+        expect(returnMessage).to.equal(`Otro personal de auxilio ha confirmado esta solicitud. (No necesita responder a este mensaje).`)
+      }
```

Comment:
I can see that you were trying to not duplicate code when you combined the Spanish and the English tests together. That was a good thing to try.

However, I'm not a big fan of this implementation. Since we're only running through two different languages and we don't ever want to add a third one to these tests (because testing two is sufficient to show that the language switching works as expected), these if/else statements make it difficult to understand at first glance what is happening here. I think you would be better off with copy-and-pasting to make sure that there is only one `expect` statement per test.